crypto/sha/sha512.c

```diff
@@ -59,6 +60,42 @@
 # define SHA512_BLOCK_CAN_MANAGE_UNALIGNED_DATA
 #endif
 
+int sha512_224_init(SHA512_CTX *c)
```

Comment:
I don't recall that lower-case would be reflection of symbol visibility... I mean I don't think there is a rule that says that unexported symbols **has to be*** lower case...

Reply:
I was in a bit of a quandary over this and other nuances.

The move seems to be towards not exporting lower level functions, thus I changed the the names to lowercase which seems to (mostly) be the convention.  I also didn't export the evp_sha512_224() function, instead only allowing access via the string representation.

I'm happy to change either or to export everything but we'll want consensus first.

Reply:
I think the EVP function should be available; the others are.  I think all-lowercase-names for internal functions is common, even if not a formal standard of ours so I would stick with that.